src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,40 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Manager extends AbstractEmployee {
+
+    private List<Employee> employees = new ArrayList<Employee>();
+
+    public Manager(String name, double salary) {
+        super(name, salary);
+    }
+
+    public abstract boolean canHireEmployee(Employee newEmployee);
+
+    void hireEmployee(Employee newEmployee) {
+        if (canHireEmployee(newEmployee))
+            getEmployees().add(newEmployee);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder manger = new StringBuilder("\t\t" + getName() + " - Manager\n");
+        for (Employee temp : getEmployees()
```

Comment:
temp tmp tmp1 tmp2 tmp3 to bardzo bardzo słabe nazwy zmiennych bo nie mówią totalnie nic nawet z niczym się nie kojarzą. W takich krótkich foreachach i lambdach najczęściej się używa jednej małej literki pierwszej litery z nazwy klasy tu by było Employee e

no i domknięcie nawiasu fora tak jak gdzieś tam wcześniej powinno być w tej samej lini